### DISCUSSION TITLE
我能列出来那些书籍的信息，但获取书籍信息出错了？

### DISCUSSION THREAD

**Discussion by 5gentle5** [Mufanc/iSmartAuto#22] [Q&A]
automaton.spider.spider:book_info:206 - [获取书籍信息] | 获取书籍信息出错：
Traceback (most recent call last):
  File "D:\iSmartAuto2-master\automaton\spider\spider.py", line 186, in book_info
    book_info = (await self.post(
KeyError: 'data'

**Comment by YBYCS** [comment]
修改booktype

**Reply by yinzeidog** [reply]
具体修改成什么呀

**Reply by YBYCS** [reply]
> 具体修改成什么呀

有没有一种可能 作者已经说过了?

**Reply by xiaoxiaoyaya** [reply]
所以怎么解决呀
